packages/block-editor/src/components/use-editor-feature/index.js

```diff
@@ -28,22 +35,23 @@ import { useBlockEditContext } from '../block-edit';
  */
 export default function useEditorFeature( featurePath ) {
 	const { name: blockName } = useBlockEditContext();
-	const path = `__experimentalFeatures.${ featurePath }`;
 
 	const setting = useSelect(
 		( select ) => {
-			const { getBlockSupport } = select( 'core/blocks' );
-
-			const blockSupportValue = getBlockSupport( blockName, path );
-			if ( blockSupportValue !== undefined ) {
-				return blockSupportValue;
-			}
```

Comment:
I don't think this hook should concern itself with `getBlockSupport`. `getBlockSupport` is only used to say that a block support a given feature and should be used directly in the "hooks" and not in `useEditorFeature` which responsibility is more related to block-editor settings and theme.json.

Reply:
This has introduced a regression by which the dropCap control is no longer shown for paragraph https://github.com/WordPress/gutenberg/pull/24930

This is also related to [this conversation](https://github.com/WordPress/gutenberg/pull/24416#issuecomment-670016243). I think we should merge the fix above so it makes it to this week's release, and then we can figure out how to settle the info coming from block.json, core's theme.json, and theme's theme.json.